app/models/driver_request.rb

```diff
@@ -0,0 +1,15 @@
+class DriverRequest < ApplicationRecord
+  belongs_to :user, required: true
+  mount_uploader :driver_licence_front, DriverIdUploader
+  mount_uploader :driver_licence_back, DriverIdUploader
+
+  validates :driver_licence_front, presence: true
+  validates :driver_licence_back, presence: true
+
+  enum stage: { requested: 0, accepted: 10, rejected: 20 }, _default: 0
+
+  scope :by_user, ->(user) { where("user_id = ?", user) }
+  scope :by_requested, ->() { where("stage = 0") }
+  scope :by_accepted, ->() { where("stage = 10") }
+  scope :by_rejected, ->() { where("stage = 20") }
```

Comment:
```ruby
scope :by_user, ->(user) { where(user_id: user.id) }
```

And here I don' remember exactly, but you definitely should be able to use your enum values in queries either by one of this two variants
```ruby
scope :by_requested, -> { where(stage: :requested) }
# OR
scope :by_requested, -> { where(stage: DriverRequest.stages[:requested]) }
```

Reply:
I was able to use enum values by calling the value as a symbol!